reports/final/4-Conclusion/0-Conclusion.md

```diff
@@ -1 +1,28 @@
-# Conclusion and Recommendations
\ No newline at end of file
+# Conclusion and Recommendations
+
+An existing issue for existing smart homes is having their system accessible enough for 
```

Comment:
Drop "existing".
Maybe use the word "problem" instead of issue
"making" instead of "having"
Drop "enough"